prime-router/src/testIntegration/kotlin/datatests/mappinginventory/TranslateAndCompareFile.kt

```diff
@@ -26,65 +26,63 @@ fun translateAndCompareHL7ToFHIR(inputFile: String, expectedOutputFile: String):
     ).runTest()
 }
 
-fun verifyHL7ToFHIRToHL7Mapping(testFileName: String): CompareData.Result {
-    val hl7ToFhirConfig = TranslationTests.TestConfig(
-        "mappinginventory/$testFileName.hl7",
-        Report.Format.HL7,
-        "",
-        "mappinginventory/$testFileName.fhir",
-        Report.Format.FHIR,
-        null,
-        true,
-        null,
-        null,
-        null
-    )
-
-    val fhirToHl7Config = TranslationTests.TestConfig(
-        "mappinginventory/$testFileName.fhir",
-        Report.Format.FHIR,
-        "",
-        "mappinginventory/$testFileName.hl7",
-        Report.Format.HL7,
-        "metadata/hl7_mapping/ORU_R01/ORU_R01-test",
-        true,
-        null,
-        null,
-        null
-    )
-
-    val hl7toFhirToHl7Config = TranslationTests.TestConfig(
-        "mappinginventory/$testFileName.hl7",
-        Report.Format.HL7,
-        "",
-        "mappinginventory/$testFileName.hl7",
-        Report.Format.HL7,
-        "metadata/hl7_mapping/ORU_R01/ORU_R01-test",
-        true,
-        null,
-        null,
-        null
-    )
-
-    val hl7ToFhirResult = TranslationTests().FileConversionTest(
-        hl7ToFhirConfig
-    ).runTest()
-
-    if (!hl7ToFhirResult.passed) {
-        return hl7ToFhirResult
+fun verifyHL7ToFHIRToHL7Mapping(
+    testFileName: String,
+    skipHl7ToFhir: Boolean = false,
```

Comment:
Doesn't look like these changes are used anywhere?  What's the use case you were envisioning?

Reply:
Ah good call, I forgot to call this out in the description. This was useful for me while trying to make sure the HL7->HL7 was working without having to worry about the fhir files being present or up-to-date. It probably is mostly for flexibility of tests during development, but could theoretically be used to have more tests that only deal with a particular combination of the input/output formats to cut back on the volume of test files we have. Definitely not closely related to the goals of this PR, so I could see a case for removing it.